### PR TITLE
changes to align the call button and mini cart

### PR DESCRIPTION
**Pull request by ReenaOmana**
Changes incorporated to align call button and mini cart

### CODE REVIEW

File: ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less [hunk 1]
```diff
@@ -1,105 +1,175 @@
...
-        text-align: center;
-		text-decoration: none;
-		width: 250px;
-		box-sizing: border-box;
-        background : #efaa9b;
-		display: inline-block;
-        &:hover{
-	  cursor: auto;
-    }
+.btn {
```
Comment (rishabh-behl) [inline ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less]: @ReenaOmana : who from UI has made these changes ?

Comment (ReenaOmana) [reply ui.apps/src/main/content/jcr_root/apps/hclecomm/components/content/button/clientlibs/less/button.less]: Ayush had made the changes.

**Review by rishabh-behl — APPROVED** [review]
LGTM

**Review by rishabh-behl — CHANGES_REQUESTED** [review]
pl resolve conflict

**Review by rishabh-behl — APPROVED** [review]
OK..

**Pull request merged**